src/style.css

```diff
@@ -1 +1,86 @@
-/* add styles here */
+:root {
+  --search-icon-big: url(/src/images/Search.svg) 26px 50% no-repeat;
+  --search-icon-small: url(/src/images/Search.svg) 13px 50% no-repeat;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__input {
+  box-sizing: border-box;
```

Comment:
It is common practice to apply the same box-sizing for all elements via `*` selector